tests/test_secret_scrubbing.py

```
@@ -14,9 +14,113 @@
+        ('secret', 'has ſecret value'),
+        ('credential', 'has credentıal value'),
+    ]
+
+    for pattern, value in isolated_examples:
+        expected = re.compile(pattern, re.IGNORECASE | re.DOTALL).search(value)
+        assert expected is not None, pattern
+
+        scrub_matches: list[logfire.ScrubMatch] = []
+
+        def callback(match: logfire.ScrubMatch):
+            scrub_matches.append(match)
```

Ruff (0.16.1)

[warning] Function definition does not bind loop variable `scrub_matches`

(B023)